docs/UserGuide.md

```diff
@@ -48,6 +48,23 @@ medication history.
 7. Refer to the [Features](#features) below for details of each command.
 
 ---
+## Glossary
+
+| Term              | Definition                                                                      |
+|-------------------|---------------------------------------------------------------------------------|
+| **Appointment**   | An arrangement to consult the doctor at a particular time.                      |
+| **Diagnosis**     | The identification of diseases by the examination of symptoms and signs.        |                                                                                                                                                                                                                                                                                    |
+| **Floor Number**  | The floor where the inpatient is on.                                            |
+| **Hospital Wing** | The section of the hospital where the inpatient is in.                          |
+| **Inpatient**     | A patient who lives in hospital while under treatment.                          |                                                                                                                                                                                                                                                                          |
```

Comment:
I would probably place inpatient and outpatient together and above floor number for consistency. Better not to use terms without defining them first.